torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -298,7 +290,9 @@ def __init__(
 
         edge_type, edge_label_index = get_edge_label_index(
             data, edge_label_index)
-
+        if edge_label is None:
```

Comment:
Will this also affect the behavior of negative sampling? Because edge_label exists, the documentation is no longer true I believe.
```
 neg_sampling_ratio (float, optional): The ratio of sampled negative
            edges to the number of positive edges.
            If :obj:`edge_label` does not exist, it will be automatically
            created and represents a binary classification task
            (:obj:`1` = edge, :obj:`0` = no edge).
            If :obj:`edge_label` exists, it has to be a categorical label from
            :obj:`0` to :obj:`num_classes - 1`.
```

Reply:
Good catch :) . maybe we set edge_label to `1` if `neg_sampling_ration` is 0 in `init`, would that resolve this?

Reply:
sounds good, thanks!